bst.rb

```diff
@@ -80,23 +80,23 @@ def search_element(root_node,value)
 
 	def delete_node(root_node,value)
 		if root_node.nil?
-		  return root_node	
+			return root_node	
 		elsif value > root_node.data
 			root_node.right = delete_node(root_node.right,value)
 		elsif value < root_node.data
 			root_node.left = delete_node(root_node.left,value)
 		else 
-		  if (root_node.left.nil? && root_node.right.nil?)
-		    return nil
-		  elsif root_node.left.nil?
-		    return root_node.right
-		  elsif root_node.right.nil?
-		    return root_node.left
-		  else
-		  	temp = smallest_element(root_node.right)
-		  	root_node.data = temp.data
-		  	root_node.right = delete_node(root_node.right,temp.data)
-		  end
+			if (root_node.left.nil? && root_node.right.nil?)
+				return nil
+			elsif root_node.left.nil?
+				return root_node.right
+			elsif root_node.right.nil?
+				return root_node.left
+			else
+				temp = smallest_element(root_node.right)
+				root_node.data = temp.data
+				root_node.right = delete_node(root_node.right,temp.data)
+			end
 		end
 		return root_node
```

Comment:
in ruby, you can skip writing return keyword if it is at end of function or block definition